File: Entities/Quiz.cs
using quiz.Entities;

public class Quiz : AuditableBaseEntity
{
    public Quiz ()
    {
        Questions = new List<Question>();
        Players = new List<Player>();
    }
    public string Pin { get; set; }
    public virtual ICollection<Question> Questions { get; set; }
    public virtual ICollection<Player> Players { get; set; }
    public bool Started { get; set; }
    public bool Ended { get; set; }
}   

File: ClientApp/src/app/models/quiz.ts
export interface Quiz {
    id:number;
    pin:string;
    players?:string[];
    questions?:Question[];
    strated?:boolean;
    ended?:boolean;
}

export interface Question{
    id:number;
    questionText:string;
    answers:Answer[];
}

export interface Answer{
    id:number;
    answerText:string;
    answerLetter?:number;
    isCorrect:false
}
